<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListWidget{
	border:2px solid #d5dce2;
	background:#f6f6f6;
	border-top-right-radius:8px;
	border-bottom-right-radius:8px;
	border-top-left-radius:8px;
	border-bottom-left-radius:8px;
}

QListWidget{
	background:#f6f6f6;
	color:black;
}

QListWidget::Item{
	border-width:0px;
	background:#ffffff;
	margin-top:1px; 
	margin-bottom:1px; 
	padding-top:10px; 
	padding-left:10px;
	padding-bottom:10px; 
	color:#454a50;
}

QListWidget::Item:hover{
	background:#49c4e6; 
}
QListWidget::item:selected{
	background:#c2e1f3; 
	color:black; 
}
QListWidget::item:selected:!active{
	background:#c2e1f3; 
	color:black; 
}

QListWidget::indicator {
	width: 40px;
	height: 40px;
}

QTextEdit{
	border:2px solid #d5dce2;
	background:#f6f6f6;
	border-top-right-radius:8px;
	border-bottom-right-radius:8px;
	border-top-left-radius:8px;
	border-bottom-left-radius:8px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="TranslatorPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="TsSrcFileNameLabel">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Source String</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="TsSrcListWidget"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="TsDstFileNameLabel">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Chinese Sim String</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="TsDstListWidget"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>171</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Source String</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="TsSrcStringLabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="TsDstToLabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Translation to 简体中文</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="TsDstStrEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="TranslationFileLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Translation File</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="SearchTsString">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/linguist/win/searchfind.png</normaloff>:/images/linguist/win/searchfind.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="shortcut">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PrevBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Previous (Up)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/linguist/win/prev.png</normaloff>:/images/linguist/win/prev.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="NextBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Next (Down)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/linguist/win/next.png</normaloff>:/images/linguist/win/next.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Down</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="prevunfinishedBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Previous Unfinish (Ctrl + Up)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/linguist/win/prevunfinished.png</normaloff>:/images/linguist/win/prevunfinished.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Ctrl+Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextunfinishedBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Next Unfinish (Ctrl + Down)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/linguist/win/nextunfinished.png</normaloff>:/images/linguist/win/nextunfinished.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Ctrl+Down</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="doneandBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Doneand (Ctrl + Enter)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/linguist/win/doneand.png</normaloff>:/images/linguist/win/doneand.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Ctrl+Alt+Return</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="doneandnextBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Doneand Next (Ctrl + Enter)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/linguist/win/doneandnext.png</normaloff>:/images/linguist/win/doneandnext.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Ctrl+Return</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="TsFileColseBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/linguist/editdelete.png</normaloff>:/images/linguist/editdelete.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionTranslation_File_ts"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionUpdate_ts"/>
    <addaction name="actionRelease_qm"/>
    <addaction name="actionExit_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionTranslation_File_ts">
   <property name="text">
    <string>&amp;Translation File (*.ts)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="actionUpdate_ts">
   <property name="text">
    <string>Update *.ts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionRelease_qm">
   <property name="text">
    <string>Release *.qm</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
